packages/mobile/src/components/drawer/Drawer.tsx

```diff
@@ -469,16 +533,19 @@ export const Drawer: DrawerComponent = ({
           if (onPercentOpen) onPercentOpen(percentOpen)
         } else if (gestureState.dy < 0) {
           // Dragging upwards
-          const percentOpen = (-1 * gestureState.dy) / drawerHeight
+          const percentOpen =
+            (-1 *
+              (isOpenIntent.current ? -1 * drawerHeight : gestureState.dy)) /
```

Comment:
could i get a little bit of understanding what this ternery is doing?

Reply:
will add a comment. essentially this makes percent bounded to [0,1] where it wasn't before. kinda an unrelated bug that got worse by sharing more animation state. overscrolling being exposed means wed do stuff like change the opacity of the play bar to <0